asana/asana.go

```diff
@@ -240,12 +240,12 @@ func (c *Client) GetUserByID(ctx context.Context, id int64, opt *Filter) (User,
 }
 
 func (c *Client) Request(ctx context.Context, path string, opt *Filter, v interface{}) error {
-	return c.request(ctx, "GET", path, nil, opt, v)
+	return c.request(ctx, "GET", path, nil, nil, opt, v)
 }
 
 // request makes a request to Asana API, using method, at path, sending data with opt filter.
 // The response is populated into v, and any error is returned.
-func (c *Client) request(ctx context.Context, method string, path string, data interface{}, opt *Filter, v interface{}) error {
+func (c *Client) request(ctx context.Context, method string, path string, data interface{}, form url.Values, opt *Filter, v interface{}) error {
```

Comment:
Now `request` has 2 ways of specifying the data. Either by setting `data`, or by setting `form`. I think it might be helpful to document that at most one can be used at the same time (or none), and that `data` takes higher precedence.

What do you think?